unity/Assets/joyrtc.cs

```diff
@@ -1,41 +1,45 @@
-using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-using Unity.WebRTC;
-using WebSocketSharp;
-
-public class CandidateData
```

Comment:
You can't change `lf` => `cr/lf`

<img width="339" alt="Screenshot 2023-08-31 at 23 00 32" src="https://github.com/binbat/joyrtc/assets/18189138/6eb217c4-992f-41e0-85a5-662a59b957d1">